DependencyInjection/Configuration.php

```diff
@@ -466,6 +467,9 @@ private function addOrmSection(ArrayNodeDefinition $node): void
                         ->end()
                         ->scalarNode('proxy_dir')->defaultValue('%kernel.cache_dir%/doctrine/orm/Proxies')->end()
                         ->scalarNode('proxy_namespace')->defaultValue('Proxies')->end()
+                        ->arrayNode('schema_ignore_classes')
```

Comment:
this needs a `->fixXmlConfig('schema_ignore_class', 'schema_ignore_classes')` on the parent node (outside the `->children()` block)